bpf/init.sh

```diff
@@ -93,8 +93,8 @@ ip addr add $HOST_IP/128 dev $HOST_DEV1
 
 ip route del $ADDR/128 dev $HOST_DEV1 2> /dev/null || true
 ip route add $ADDR/128 dev $HOST_DEV1
-ip route del $ADDR/112 via $ADDR 2> /dev/null || true
-ip route add $ADDR/112 via $ADDR
+ip route del $ADDR/96 via $ADDR 2> /dev/null || true
+ip route add $ADDR/96 via $ADDR
```

Comment:
@tgraf This lines seems to work but after checking the addresses in the interface I don't see the address there, do you know why?

```
2037: cilium_host@cilium_net: <BROADCAST,MULTICAST,NOARP,UP,LOWER_UP> mtu 1500 state UP qlen 1000
    inet6 fd00::c0a8:210c:0:ffff/128 scope global 
       valid_lft forever preferred_lft forever
    inet6 fe80::6439:94ff:fea1:893d/64 scope link 
       valid_lft forever preferred_lft forever
```